### DISCUSSION TITLE
How can I hide the menu-bar icon

### DISCUSSION THREAD

**Discussion by abrahamnm** [will-stone/browserosaurus#569] [Q&A] [answered]
It would be great if you could hide the menu-bar icon. I hardly use it.

**Comment by pressdarling** [comment, marked as answer]
This appears to be a duplicate of #439 which was addressed by @will-stone - see this answer [there](https://github.com/will-stone/browserosaurus/discussions/439#discussioncomment-1270431).

I use Bartender 4 for this, but the other listed alternatives may work for you and will take care of other menu bar icons too. Removing the icon will remove the primary way of interacting with the app.
